### PR TITLE
Transform support

### PR DESCRIPTION
**Pull request by kmelmon**
This change adds support for setting the transform property.  The change is very simple as the javascript code already multiplies all the transform components together into a single matrix and sends that over the bridge.  This code just recognizes the property and assigns the matrix to the UIElement.TransformMatrix property.

Note that this implementation will only work on RS5+ as this is the most performant way to set the TransformMatrix into XAML.  When we implement the native driver it will likely use RS5+ animation features as well.  If we see the need to polyfill we can address that in a follow-up change.

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/react-native-windows/pull/2513)

### CODE REVIEW

File: vnext/ReactUWP/Views/FrameworkElementViewManager.cpp [hunk 1]
```diff
@@ -95,6 +96,38 @@ void FrameworkElementViewManager::UpdateProperties(ShadowNodeBase* nodeToUpdate,
           continue;
         }
       }
+      if (propertyName == "transform")
```
Comment (ahimberg) [inline vnext/ReactUWP/Views/FrameworkElementViewManager.cpp]: else if

Comment (khetanashita) [inline vnext/ReactUWP/Views/FrameworkElementViewManager.cpp]: Does the property need to be added in TransferProperties() as well?

Comment (kmelmon) [reply vnext/ReactUWP/Views/FrameworkElementViewManager.cpp]: fixed

Comment (kmelmon) [reply vnext/ReactUWP/Views/FrameworkElementViewManager.cpp]: good catch, fixed

**Pull request merged**